main/ui.js

```diff
@@ -22,7 +22,6 @@ module.exports = async function (ctx) {
     height: store.get('ui.height', dimensions.height < 900 ? dimensions.height : 900),
     minWidth: 720,
     minHeight: 540,
-    autoHideMenuBar: true,
```

Comment:
Are you sure we want to show the menu bar? I think the idea was to fully control Station's window to provide consistent design & interaction patterns and also have the same UI on all platforms.

Mixing Windows-native menubar rendered with built-in Windows fonts and Electron-rendered screens with our custom fonts is going to look ugly IMO.

Maybe this is a hint that we should re-think our current usage of the OS menu bar and implement our custom in-app menu instead.

Anyhow, I am ok to iterate on this. We can land your pull request as it is now.

Reply:
I agree this is not a great experience, but we haven't yet designed a better one. Maybe we can figure something out until release, but for now this is will get the functional side of things done. Or we could say that we rather have a cleaner window look and ditch the items "Check for updates" and "Save Saturn Module Log As..." (or we could move them to the tray). What do you think?

Reply:
My preference is to have a cleaner window look. I like to idea of adding "Save Saturn Module Log As..." to the Tray menu. I believe "Check for updates" is already there :)

Reply:
good point, then I'll move the log item there and disable the menus again 👍

Reply:
See https://github.com/filecoin-project/filecoin-station/blob/bd89b53f91a4cbec4b1c60fd2ef82bc3f3522d89/main/tray.js#L29-L39

I'll leave the decision up to you.

If you decide to show the menu bar, then please create a follow-up GH issue to remind us to revisit this part of the design.